jepsen/src/jepsen/nemesis.clj

```diff
@@ -14,25 +14,20 @@
     (invoke! [this test op] op)
     (teardown! [this test] this)))
 
-(defn snub-node!
-  "Drops all packets from node."
-  [node]
-  (c/su (c/exec :iptables :-A :INPUT :-s (net/ip node) :-j :DROP)))
-
 (defn snub-nodes!
   "Drops all packets from the given nodes."
-  [nodes]
-  (dorun (map snub-node! nodes)))
+  [test dest sources]
+  (doseq [src sources] (net/drop! (:net test) test src dest)))
```

Comment:
OK I'm just gonna ask for oonnnnne more thing here, that I didn't think of before--we should still parallelize the cutoffs by writing this as (->> sources (pmap #(net/drop! (:net test) test % dest)) dorun)